packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx

```diff
@@ -164,25 +185,18 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
         return (selected_list_category || list_with_category()[0]).contract_categories;
     };
 
-    const selected_contract_index = () => {
```

Comment:
@henry-deriv many changes in this file are also related to Carousel removal which is not related to this PR. Is it? :)